qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -449,7 +476,11 @@ bool HOCRPdfExporter::run(QString& filebasename) {
 	pdfSettings.uniformizeLineSpacing = ui.checkBoxUniformizeSpacing->isChecked();
 	pdfSettings.preserveSpaceWidth = ui.spinBoxPreserve->value();
 	pdfSettings.overlay = ui.comboBoxOutputMode->currentIndex() == 1;
-	pdfSettings.detectedFontScaling = ui.spinFontScaling->value() / 100.;
+	pdfSettings.detectedFontScaling = ui.spinFontScaling->value() / 100.0;
+	pdfSettings.paperSize = ui.comboBoxPaperSize->itemData(ui.comboBoxPaperSize->currentIndex()).toString();
+
+	PaperSizeMetrics currentMetric = static_cast<PaperSizeMetrics>(ui.comboBoxPaperSizeMetric->itemData
+			(ui.comboBoxPaperSizeMetric->currentIndex()).toInt());
```

Comment:
It is not necessary to store the paperSize in the pdfSettings, since as far as I can see it is only used in this scope.

Just compute the inch size once here instead of doing it again and again for each page below.

Reply:
Fixed.